stats/out_graphite.go

```diff
@@ -105,6 +108,7 @@ func (g *Graphite) writer() {
 		var ok bool
 		for !ok {
 			conn = assureConn()
+			conn.SetDeadline(time.Now().Add(g.timeout))
```

Comment:
I think this needs to just be `conn.SetWriteDeadline()`.   Calling SetDeadline() will cause the checkEOF() read to timeout

Reply:
right, updated